### PR TITLE
v5.5.0

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,59 @@
...
+- 🌍 Add Danish (daDK) locale (#3800) @kasperfilstrup
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.5.0` / `@mui/x-data-grid-pro@v5.5.0`
+
+
+#### Changes
+
+- [DataGrid] Add methods to import and export the state (#3593) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:25]: We are not advertising about that one yet
We will uncomment the doc and put it in the highlight once the column size and order are portable (see https://github.com/mui-org/material-ui-x/pull/3593#pullrequestreview-870002619)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,59 @@
...
 
+## 5.5.0
+
+_Feb 3, 2022_
+
+A big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add support for master/detail (#3387) @m4theushw
+
+  TODO: Add description + example
```
Comment (flaviendelangle) [inline CHANGELOG.md:14]: @m4theushw

**Review by m4theushw — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,58 @@
...
+- 🌍 Add Danish (daDK) locale (#3800) @kasperfilstrup
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.5.0` / `@mui/x-data-grid-pro@v5.5.0`
+
+#### Changes
+
+- [DataGrid] Add methods to import and export the state (#3593) @flaviendelangle
+- [DataGrid] Fix <kbd>PageUp</kbd> jumping directly to the column header skipping the first row (#3761) @cherniavskii
```
Comment (DanailH) [inline CHANGELOG.md:25]:
```suggestion
- [DataGrid] Fix `<kbd>PageUp</kbd>` jumping directly to the column header skipping the first row (#3761) @cherniavskii
```

**Review by DanailH — APPROVED** [review]
I thought we will release the master/detail panel as well or maybe I misunderstood?

File: CHANGELOG.md (hunk 3 above)
Comment (flaviendelangle) [reply CHANGELOG.md:25]: image


The others keys don't have it :angel

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,17 +9,16 @@ _Feb 3, 2022_
...
-  TODO: Add description + example
+  <img src="https://user-images.githubusercontent.com/42154031/152379354-47120aac-2b37-4a90-b311-64b4522283b9.gif" width="814">
 
 - 🌍 Add Danish (daDK) locale (#3800) @kasperfilstrup
 - 📚 Documentation improvements
 - 🐞 Bugfixes
 
 ### `@mui/x-data-grid@v5.5.0` / `@mui/x-data-grid-pro@v5.5.0`
 
-
```
Comment (m4theushw) [inline CHANGELOG.md]: @alexfauquette The changelog script is adding two newlines.

File: CHANGELOG.md [hunk 5]
```diff
@@ -333,12 +381,7 @@ A big thanks to the 5 contributors who made this release possible. Here are some
 - 🙈 Add a new option to hide the amount of descendant on the grouping cells of the Tree Data (#3368) @flaviendelangle
 
   ```tsx
-  <DataGridPro
-    treeData
-    rows={rows}
-    columns={columns}
-    groupingColDef={{ hideDescendantCount }}
-  />
+  <DataGridPro treeData rows={rows} columns={columns} groupingColDef={{ hideDescendantCount }} />
```
Comment (alexfauquette) [inline CHANGELOG.md:384]: I think your prettier has modified some preview changelog

File: CHANGELOG.md (hunk 3 above)
Comment (m4theushw) [reply CHANGELOG.md:25]: The backticks override the kbd formatting: `<kbd>PageUp</kbd>` vs <kbd>PageUp</kbd>.

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (flaviendelangle) [reply CHANGELOG.md:384]: It was @m4theushw commit but yes I suppose an update on Prettier has changed some of the behaviors

File: CHANGELOG.md (hunk 4 above)
Comment (alexfauquette) [reply CHANGELOG.md]: I prefer to say: "the prettier does not remove the two line breaks" ;)

The correction: #3833

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,58 @@
...
+#### Changes
+
+- [DataGrid] Add methods to import and export the state (#3593) @flaviendelangle
+- [DataGrid] Fix <kbd>PageUp</kbd> jumping directly to the column header skipping the first row (#3761) @cherniavskii
+- [DataGrid] Throw an error if incorrect column type is used (#3757) @DanailH
+- [DataGridPro] Add support for master/detail (#3387) @m4theushw
+- [l10n] Add Danish (daDK) locale (#3800) @kasperfilstrup
+- [l10n] Improve Dutch (nlNL) locale (#3724) @MatthijsKok
+- [l10n] Improve Hebrew (heIL) locale (#3775) @ColdAtNight
+- [l10n] Improve Russian (ruRU) locale (#3818) @Leniorko
```
Comment (oliviertassinari) [inline CHANGELOG.md:31]: Would it make more sense to use the ISO name (BCP 47 language tag), than the code export name:

```suggestion
- [l10n] Add Danish (da-DK) locale (#3800) @kasperfilstrup
- [l10n] Improve Dutch (nl-NL) locale (#3724) @MatthijsKok
- [l10n] Improve Hebrew (he-IL) locale (#3775) @ColdAtNight
- [l10n] Improve Russian (ru-RU) locale (#3818) @Leniorko
```

Comment (flaviendelangle) [reply CHANGELOG.md:31]: Some users use the `xxXX` format, other the `xx-XX` (the BCP 47 language tag is fancier :laughing: )
I have no preference here

Comment (oliviertassinari) [reply CHANGELOG.md:31]: For context, I was coming from this background: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

```jsx
console.log(new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(number));
```
😁
The `heIL` seems to be more a convention that MUI has to export the locale, than the actual local

Comment (cherniavskii) [reply CHANGELOG.md:31]: I agree, `xxXX` is just an export name.
I'll update to `xx-XX` in https://github.com/mui/mui-x/pull/3914

File: packages/x-license-pro/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "5.4.0",
+  "version": "5.5.0",
```
Comment (oliviertassinari) [inline packages/x-license-pro/package.json:3]: Something looks broken here. There were no changes in this package https://github.com/mui/mui-x/compare/v5.4.0...v5.5.0. I would expect no diffs

```diff
  "version": "5.4.0",
```

**Pull request merged**